### PR TITLE
bug : date picker is showing default date 1880 in date range

### PR DESCRIPTION
**Pull request by johibkhan2**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/webapp/app/entities/flair-bi/filter/filter-date-range.component.js [hunk 1]
```diff
@@ -219,7 +219,7 @@
                 return null;
             }
             // console.log('str to date', date, typeof date);
-            return new Date(date);
+            return Date.parse(date) ? new Date(date) : null;
```
Comment (ir73) [inline src/main/webapp/app/entities/flair-bi/filter/filter-date-range.component.js:222]: is there any scenario when `Date.parse(date)` won't return a result, but `new Date(date)`? Just curious if `new Date(date)` is needed at all here.

Comment (johibkhan2) [reply src/main/webapp/app/entities/flair-bi/filter/filter-date-range.component.js:222]: this(Date.parse(date)) is just to check whether date is valid or not.
when select the dynamic date range and select the day . datepicker shows the calander of 1880 as you assign the "_FLAIR_INTERVAL_OPERATION(NOW()" to date field...it returns the "invalid date" during conversion

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Pull request merged**